app/views/admin/settings/show.html.erb

```diff
@@ -23,14 +23,13 @@
           class="crayons-btn crayons-btn--secondary float-right"
           type="button"
           data-action="click->config#toggleAccordionButtonLabel">
-
-          <%= VerifySetupCompleted.setup_completed? ? "Show Getting Started" : "Hide Getting Started" %>
+          Hide Getting Started
         </button>
       </div>
 
       <div
         id="getStartedBodyContainer"
-        class="<%= VerifySetupCompleted.setup_completed? ? "hide collapse" : "show expand" %> p-3"
+        class="p-3"
```

Comment:
@juliannatetreault should we not explicitly set the hide/show class here on the "getStartedBodyContainer"? It seems to be the case for other containers. Also, perhaps in this one or a follow up PR we can remove this entire section completely, as it doesn't really make sense anymore or correspond with our onboarding section.

Reply:
Great point; I will remove this entirely in a follow-up PR so that we can get this work shipped, @Ridhwana!